python/requirements.txt

```diff
@@ -10,7 +10,6 @@ aioredis
 click >= 7.0
 cloudpickle
 colorama
-colorful
```

Comment:
is there a separate PR for colorful?

Reply:
yep! we already did this a couple weeks ago.